src/Microsoft.Performance.SDK/Processing/Projection/ClipTimeToViewportProjection.cs

```diff
@@ -158,20 +158,20 @@ private struct ClipTimeToViewportTimeRangeColumnGenerator<TGenerator>
             {
                 internal TGenerator Generator { get; }
 
-                internal VisibleTableRegionContainer ViewPortContainer { get; }
+                private readonly VisibleTableRegionContainer viewportContainer;
 
                 public ClipTimeToViewportTimeRangeColumnGenerator(TGenerator timestampGenerator)
```

Comment:
>timestampGenerator [](start = 77, length = 18)

not part of this PR, but if you have any other changes to make, please consider renaming this to timeRangeGenerator :)